graylog2-web-interface/src/views/logic/searchtypes/messages/MessageSortConfig.js

```diff
@@ -0,0 +1,36 @@
+// @flow strict
+import Direction from 'views/logic/aggregationbuilder/Direction';
+import SortConfig, { Builder } from 'views/logic/aggregationbuilder/SortConfig';
+
+export type MessageSortConfigJson = {
+  field: string,
+  order: 'ASC' | 'DESC',
+};
+
+export default class MessageSortConfig {
+  _value: SortConfig
+
+  constructor(type: string, field: string, direction: Direction) {
+    this._value = new SortConfig(type, field, direction);
```

Comment:
I think there is no benefit reusing `SortConfig` here. Instead, we could just store the `field`/`direction` values in internal fields (we do not even need getters for them) and just define the JSON (de-)serialization format in here.